vkconfig/configurator.cpp

```diff
@@ -757,57 +757,99 @@ const QString &Configurator::GetCustomLayersPath(int path_index) const {
 void Configurator::SelectLaunchApplication(int application_index) {
     if (application_index < 0) return;
 
-    _active_launch_executable_path = _overridden_application_list[application_index]->executable_path;
+    _active_launch_executable_path = _overridden_applications[application_index]->executable_path;
 }
 
 int Configurator::GetLaunchApplicationIndex() const {
-    for (int i = 0; i < _overridden_application_list.size(); i++) {
-        if (_overridden_application_list[i]->executable_path == _active_launch_executable_path) return i;
+    for (int i = 0; i < _overridden_applications.size(); i++) {
+        if (_overridden_applications[i]->executable_path == _active_launch_executable_path) return i;
     }
 
     return 0;  // Not found, but the list is present, so return the first item.
 }
 
+static QString GetDefaultExecutablePath(const QString &executable_name) {
+#if defined(__APPLE__)
```

Comment:
You may run into merge problems here as the last PR replaced these #def's.

Reply:
I expect I will yes